.github/workflows/ci.yml

```diff
@@ -129,39 +129,39 @@ jobs:
     name: Build Windows
     uses: ./.github/workflows/build-native.yml
     with:
-     runner: windows
+     runner: windows-2022
```

Comment:
why do we need to fixate the windows version?

Reply:
Currently `windows-latest` is the same as `windows-2022`. I decided to fix it in case the latest image of windows is changed.

Reply:
We should be using the latest image unless we really need to use a specific version